### ISSUE TITLE
Duplicate headers

### ISSUE THREAD

**Issue by abhinav** [yarpc/yarpc#21]
When converting from HTTP headers to YARPC headers, there's a possibility that the same header name has multiple values associated with it. We should pick a conflict resolution strategy and use it consistently across all languages.

**Comment by abhinav** [comment]
@yarpc/yarpc

**Comment by abhinav** [comment]
@yarpc/yarpc

**Comment by breerly** [comment]
@abhinav now that we're normalizing HTTP headers, this should be done right?

**Comment by abhinav** [comment]
No. The behavior is still undefined for multiple occurrences of the same header. We'll need to pick a resolution strategy first.

**Comment by abhinav** [comment]
A simple resolution strategy that we could go for: The last occurrence wins.

**Comment by breerly** [comment]
I'm on board with that.

**Comment by kriskowal** [comment]
So say we all.
